inc/vr_update.hpp

```diff
@@ -114,6 +114,10 @@ extern "C"
 #define SLEEP_1               (1000000)
 #define SLEEP_2               (2000000)
 #define SLEEP_1000            (1000)
+#define INDEX_40              (0x40)
```

Comment:
Is this code improvement or  c+23 is not happy with multiple places defeinition?

Reply:
Looks like it is bug . better to fix using different commit. use version change specific commits in this.